### PR TITLE
Add Rollbar error handling

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description
The goal of this PR is to introduce [Rollbar](https://rollbar.com/) error tracking.

##### Checklist
- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

File: app/app/local_settings.py.dist [hunk 1]
```diff
@@ -117,6 +118,10 @@ GITTER_TOKEN = '' #https://developer.gitter.im/docs/authentication
 # optional: only needed if you slack things
 SLACK_TOKEN = 'TODO'
 
+# Rollbar tokens
+ROLLBAR_CLIENT_TOKEN = os.environ.get('ROLLBAR_CLIENT_TOKEN', '')
```
Comment (owocki) [inline app/app/local_settings.py.dist]: youll want to make a similar update to the production local settings for this.

File: app/app/context.py [hunk 2]
```diff
@@ -14,5 +14,7 @@ def insert_settings(request):
         'trackjs_token': settings.TRACKJS_TOKEN,
```
Comment (owocki) [inline app/app/context.py]: we should lose trackJS ... that is the old tool used to track production JS errors.

**Comment by owocki** [comment]
left a comment or two.. mostly LGTM

File: app/app/local_settings.py.dist (hunk 1 above)
Comment (owocki) [reply app/app/local_settings.py.dist]: whats best practice for local development?  should we all have our own rollbar tokens?

Comment (mbeacom) [reply app/app/local_settings.py.dist]: For local development, I was leaving that up to the individual.  If the tokens (`ROLLBAR_SERVER_TOKEN`/`ROLLBAR_CLIENT_TOKEN`) aren't provided, rollbar is disabled by default.  If that will suffice, I'll update the documentation to reflect that.

Comment (owocki) [reply app/app/local_settings.py.dist]: works for me

**Comment by thelostone-mc** [comment]
Did not know about this :O shit rollbar looks pretty sweet

**Pull request merged**